.github/workflows/iroha2.yml

```diff
@@ -50,7 +50,7 @@ jobs:
         with:
           context: .
           push: true
-          tags: hyperledger/iroha2:dev
+          tags: hyperledger/iroha2:2.0.0-pre.rc2
```

Comment:
Since we already have the notation `2.0.0-pre-rc.2`, why don't we unify one or the other?